observability/logging/logging-6.2/6x-cluster-logging-deploying-6.2.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[id="installing-logging-6-2"]
+= Installing Logging
+:context: installing-logging-6-2
+
+toc::[]
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user through the CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
```

Comment:
```suggestion
{product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user through the CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the logic of the Operator. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
```

![Screenshot From 2025-03-10 13-38-02](https://github.com/user-attachments/assets/919d595e-b503-42da-8d76-c7e0c820eae6)


{product-title} or {Product-title}?

Reply:
Might be out of scope for your PR.